modules/arxiv.py

```diff
@@ -85,10 +89,10 @@ def summary(jenni, input):
     else:
         clipped = long_summary
 
-    jenni.say(clipped)
+    return jenni.say(clipped)
```

Comment:
i had assumed it was returning some positive status, since it was all over the place, but i guess it was just short-hand for jenni.say(...); return